src/SmartComponents/PackageDetail/PackageDetail.test.js

```diff
@@ -41,6 +42,7 @@ const initStore = (state) => {
     return mockStore({ PackageDetailStore: state });
 };
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
Please, don't do this. Correct what is not right and don't ignore it.

Reply:
You are right, removed all of these exceptions